package sistemaCineTrack;

public class Avaliacao {

    private int id;
    private int nota;
    private String comentario;

    public Avaliacao(int id, int nota, String comentario) {

        this.id = id;
        this.nota = nota;
        this.comentario = comentario;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getNota() {
        return nota;
    }

    public void setNota(int nota) {
        this.nota = nota;
    }

    public String getComentario() {
        return comentario;
    }

    public void setComentario(String comentario) {
        this.comentario = comentario;
    }

    @Override
    public String toString() {
        return "ID: " + id +
                " | Nota: " + nota +
                " | Comentário: " + comentario;
    }
}